shardingsphere-sql-parser/shardingsphere-sql-parser-test/src/main/resources/case/dml/select-pagination-group-by-order-by.xml

```diff
@@ -128,991 +166,1364 @@
                 </subquery>
             </subquery-table>
         </from>
         <where start-index="351" stop-index="372" literal-start-index="352" literal-stop-index="373">
-            <and-predicate>
-                <predicate start-index="357" stop-index="372" literal-start-index="358" literal-stop-index="373">
-                    <column-left-value start-index="357" stop-index="368" literal-start-index="358" literal-stop-index="369" name="rownum_">
-                        <owner start-index="357" stop-index="360" literal-start-index="358" literal-stop-index="361" name="row_" />
-                    </column-left-value>
-                    <operator type="&gt;" />
-                    <compare-right-value>
-                        <parameter-marker-expression start-index="372" stop-index="372" value="5" />
-                        <literal-expression start-index="373" stop-index="373" value="6" />
-                    </compare-right-value>
-                </predicate>
-            </and-predicate>
+            <expr>
+                <binary-operation-expression start-index="357" stop-index="372" literal-start-index="358" literal-stop-index="373">
+                    <left>
+                        <column name="rownum_" start-index="357" stop-index="368" literal-start-index="358" literal-stop-index="369">
+                            <owner name="row_" start-index="357" stop-index="360" literal-start-index="358" literal-stop-index="361" />
+                        </column>
+                    </left>
+                    <operator>&gt;</operator>
+                    <right>
+                        <literal-expression value="6" start-index="373" stop-index="373" />
+                        <parameter-marker-expression value="5" start-index="372" stop-index="372" />
+                    </right>
+                </binary-operation-expression>
+            </expr>
         </where>
     </select>
 
-    <select sql-case-id="select_pagination_with_top_and_diff_group_by_and_order_by_and_parentheses" parameters="3, 1, 2, 9, 10, 6">
+    <select sql-case-id="select_pagination_with_top_and_diff_group_by_and_order_by_and_parentheses"
+            parameters="3, 1, 2, 9, 10, 6">
         <projections start-index="7" stop-index="7">
-            <shorthand-projection start-index="7" stop-index="7" />
+            <shorthand-projection start-index="7" stop-index="7"/>
         </projections>
         <from>
             <subquery-table alias="row_">
                 <subquery>
                     <select>
                         <projections start-index="22" stop-index="160">
                             <top-projection alias="rownum_" start-index="22" stop-index="72">
-                                <top-value value="3" parameter-index="0" start-index="27" stop-index="27" />
+                                <top-value value="3" parameter-index="0" start-index="27"
+                                           stop-index="27"/>
```

Comment:
@lwtdev Like this, is it better to merge them into one line?

Reply:
@jingshanglu OK, I will reformat them.